doc/api/next_api_changes/deprecations/20063-AL.rst

```diff
@@ -0,0 +1,3 @@
+``Divider.get_vsize_hsize`` and ``Grid.get_vsize_hsize``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+These ``axes_grid1`` methods are deprecated.
```

Comment:
Worth adding a quick ref to `Divider.get_vertical()` and `Divider.get_horizontal()`?

Reply:
Added a suggestion to copy the implementation if needed, instead of trying to explain it with words, which would be longer...